### PR TITLE
better handling booleans and strings without default values when gene…

### PR DESCRIPTION
**Pull request by alon-dotan-starkware**
…rating conf file

### CODE REVIEW

File: configargparse.py [hunk 1]
```diff
@@ -673,7 +676,7 @@ def get_command_line_key_for_unknown_config_file_setting(self, key):
 
         return command_line_key
 
-    def get_items_for_config_file_output(self, source_to_settings,
+def get_items_for_config_file_output(self, source_to_settings,
```
Comment (bw2) [inline configargparse.py]: looks like accidentally deleted indent

Comment (bw2) [reply configargparse.py]: also please make sure all unittests pass
python3 -m unittest tests.test_configargparse
https://travis-ci.org/github/bw2/ConfigArgParse/builds/759318560

**Review by bw2 — APPROVED** [review]

File: configargparse.py (hunk 1 above)
Comment (alon-dotan-starkware) [reply configargparse.py]: Done.

**Review by bw2 — CHANGES_REQUESTED** [review]
Some tests still failing
https://travis-ci.org/github/bw2/ConfigArgParse/builds/759326686

File: configargparse.py (hunk 1 above)
Comment (alon-dotan-starkware) [reply configargparse.py]: looks like I will need some assist to better understand the test

**Comment by bw2** [comment]
Closing due to failing tests.

**Pull request closed without merging**